doc-content/enterprise-only/openshift/operator-deploy-start-proc.adoc

```diff
@@ -11,7 +11,7 @@ To start deploying a {PRODUCT} environment using {operator_name}, access the ins
 
 . In the {OPENSHIFT} web cluster console menu, select *Catalog -> Installed operators* or *Operators -> Installed operators*.
 . Click the name of the operator that contains `businessautomation`. Information about this operator is displayed.
-. Click the *Installer* link, located on the left side of the window in {OPENSHIFT} version 4.1 or on the right side of the window in {OPENSHIFT} version 4.2 or later.
+. Click the *Installer* link, located on the right side of the window.
```

Comment:
I'd suggest changing this sentence to:

 Click *Installer*.

and if it is necessary to provide the location
then 

Click *Installer* link available on the right side.